.travis.yml

```diff
@@ -1,13 +1,62 @@
 language: php
 
+services:
+  - mysql
+
```

Comment:
I'm not so good in Travis CI, so I have a question: is it possible to run tests on both MySQL and PostgreSQL. It would be great to support both engines without any errors.